openshift-examples/keycloak.yaml

```diff
@@ -70,7 +70,7 @@ objects:
                   value: dns.DNS_PING
                 - name: JGROUPS_DISCOVERY_PROPERTIES
                   value: 'dns_query=${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local'
-              image: quay.io/keycloak/keycloak:18.0.0-SNAPSHOT
+              image: quay.io/keycloak/keycloak:999-SNAPSHOT
```

Comment:
```suggestion
              image: quay.io/keycloak/keycloak:latest
```

Reply:
Setting to nightly in https://github.com/keycloak/keycloak-quickstarts/pull/310